auslib/test/admin/views/test_releases.py

```diff
@@ -783,3 +800,68 @@ def testSettings(self):
         ret = self._get('/releases/settings')
         self.assertStatusCode(ret, 200)
         self.assertEqual(json.loads(ret.data), json.loads(data['blob']))
+
+
+class TestReadOnlyView(ViewTest, JSONTestMixin):
+
+    def testReadOnlyGet(self):
+        ret = self._get('/releases/b/read_only')
+        is_read_only = dbo.releases.t.select(dbo.releases.name == 'b').execute().first()['read_only']
+        self.assertEqual(json.loads(ret.data)['read_only'], is_read_only)
+
+    def testReadOnlySetTrueAdmin(self):
+        data = dict(name='b', read_only=True, product='Firefox', data_version=1)
+        self._put('/releases/b/read_only', username='bill', data=data)
+        read_only = dbo.releases.isReadOnly(name='b')
+        self.assertEqual(read_only, True)
```

Comment:
Turns out this is PEBKAC! These tests are fine, my manual tests were using the wrong endpoints. Ignore this!